### PR TITLE
1483 course preview about course

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Course preview

### Changes proposed in this pull request
- Setup course preview contents nav
- Add about course content
- Add markdown and a nice parser from [GDS RE team](https://github.com/alphagov/re-rotas/blob/master/app/lib/rotas/markdown_renderer.rb)

### Guidance to review
Example - `/organisations/2AT/courses/3CXF/preview`

# After
localhost_3000_organisations_2AT_courses_3CXF_preview(Laptop with MDPI screen)

### CODE REVIEW

File: spec/lib/govuk/markdown_renderer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+# Ignore whitespace
+def expect_equal_ignoring_ws(first, second)
+  expect(first.lines.map(&:strip).join('')).to eq(second.lines.map(&:strip).join(''))
+end
+
+describe Govuk::MarkdownRenderer do
+  let(:html) { Govuk::MarkdownRenderer.render(markdown) }
+
+  context 'ul' do
```
Comment (misaka) [inline spec/lib/govuk/markdown_renderer_spec.rb]: Could we make these `describe` please? As-in, this is the subject of the test.

File: spec/lib/govuk/markdown_renderer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    let(:markdown) do
+      <<~MD
+        - item
+        - item
+        - item
+      MD
```
Comment (misaka) [inline spec/lib/govuk/markdown_renderer_spec.rb]: The specific example of a list we saw today was a list that followed a paragraph with no line-break. Can we test that we render that correctly here too?

Comment (fofr) [reply spec/lib/govuk/markdown_renderer_spec.rb]: When I spoke to @dankmitchell he said he would look at this in a follow-up PR.

File: spec/lib/govuk/markdown_renderer_spec.rb (hunk 1 above)
Comment (dankmitchell) [reply spec/lib/govuk/markdown_renderer_spec.rb]: Done

**Pull request merged**